.circleci/config.yml

```diff
@@ -68,22 +71,21 @@ jobs:
           command: |
             cd examples/vue-kitchen-sink
             yarn build-storybook
-
       - run:
-          name: "Run react kitchen-sink"
+          name: "Run react kitchen-sink (smoke test)"
           command: |
             cd examples/cra-kitchen-sink
-            yarn storybook
-          background: true
+            yarn storybook --smoke-test
       - run:
-          name: "Run vue kitchen-sink"
+          name: "Run vue kitchen-sink (smoke test)"
           command: |
             cd examples/vue-kitchen-sink
-            yarn storybook
-          background: true
+            yarn storybook --smoke-test
       - run:
-          name: Workaround for https://github.com/GoogleChrome/puppeteer/issues/290
-          command: sh ./scripts/workaround-puppeteer-issue-290.sh
+          name: "Run image snapshots on react kitchen-sink"
+          command: |
+            cd examples/cra-kitchen-sink
+            yarn test-image-snapshots
```

Comment:
Please make it available as `yarn test --image` in root. You need to edit `scripts/test.js` to achieve that. Then add a note about it to `CONTRIBUTING.md`

Reply:
sure ! Thanks